src/librustc/middle/lang_items.rs

```diff
@@ -90,6 +90,10 @@ impl LanguageItems {
         }
     }
 
+    pub fn require_owned_box(&self) -> Result<ast::DefId, String> {
```

Comment:
Other places that do this sort of thing just write the `self.require(...)` call inline, maybe it's ok to do that here too instead of introducing a special method?